From 844062c0b008d5221e89a042238421d8fa1af45e Mon Sep 17 00:00:00 2001
From: Martin Wilck <mwilck@suse.com>
Date: Wed, 11 Mar 2020 23:41:33 +0100
Subject: [PATCH 126/130] Fix GNOME shell error message about factsBox.FactsBox

Fixes this error message:

gnome-shell[13339]: Some code accessed the property 'FactsBox' on the module
'factsBox'. That property was defined with 'let' or 'const' inside the
module. This was previously supported, but is not correct according to the ES6
standard. Any symbols to be exported from a module must be defined with
'var'. The property access will work as previously for the time being, but
please fix your code anyway.
---
 extension/widgets/factsBox.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/extension/widgets/factsBox.js b/extension/widgets/factsBox.js
index e524781..68c74b9 100644
--- a/extension/widgets/factsBox.js
+++ b/extension/widgets/factsBox.js
@@ -43,7 +43,7 @@ const TodaysFactsWidget = Me.imports.widgets.todaysFactsWidget.TodaysFactsWidget
  * well as todays facts.
  * @class
  */
-const FactsBox = GObject.registerClass(
+var FactsBox = GObject.registerClass(
 class FactsBox extends PopupMenu.PopupBaseMenuItem {
     _init(controller, panelWidget) {
         super._init({reactive: false});
-- 
2.26.1

